lib/kiba-common/transforms/source_transform_adapter.rb

```diff
@@ -3,7 +3,14 @@ module Common
     module Transforms
       class SourceTransformAdapter
         def process(args)
-          args.shift.new(*args).each do |row|
+          klass = args.shift
```

Comment:
`args.shift` is modifying the original array. It would be better to avoid doing so.